packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -501,6 +492,19 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
             }
         }
 
+        if (!this.odspSummaryUploadManager) {
+            const module = await import("./internalModule");
```

Comment:
I assume it's safe to have concurrent import() calls, right?

Reply:
Yes, it's safe. If this call is very high frequency, we could locally cache that promise and locally cache the result once it resolves for the best performance possible.